examples/mysql-backup-create-service-account/versions.tf

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_version = ">=v1.1.6"
```

Comment:
Are reason why we need >v1.1.6 or can we use >= 0.13?

Reply:
Just not tested with anything older that's the reason. Will adapt if this is not an issue

Reply:
Could you try a `terraform validate` with 0.13. We prefer to keep version constraints aligned with them module constraints.

Reply:
This works so will downgrade to 0.13 min requirement